.gitignore

```diff
@@ -30,11 +30,6 @@ yarn-debug.log*
 yarn-error.log*
 .pnpm-debug.log*
 
-# local env files
-# do not commit any .env files to git, except for the .env.example file. https://create.t3.gg/en/usage/env-variables#using-environment-variables
-.env
-.env*.local
-
```

Comment:
Hmm... Is this intentional?

Reply:
Yeah, we have no secrets in `.env` and without it someone couldn't get this project running with a simple clone. Normally environmental variables which are sensitive come from a secrets store like Vault.